index.bs

```diff
@@ -760,6 +760,21 @@ To <dfn>find a matching trigger spec</dfn> given an [=attribution source=]
 
     </dl>
 
+<h3 dfn-type=dfn>Aggregate debug reporting config</h3>
+
+An aggregate debug reporting config is a [=struct=] with the following items:
+
+<dl dfn-for="aggregate debug reporting config">
+: <dfn>key piece</dfn> (default 0)
+:: A non-negative 128-bit integer.
+: <dfn>debug data</dfn> (default an [=map/is empty|empty=] [=map=])
+:: A [=map=] whose [=map/keys] are [=debug data types=] and whose [=map/values=]
+    are [=aggregatable contributions=].
+: <dfn>aggregation coordinator</dfn> (default [=default aggregation coordinator=])
+:: An [=aggregation coordinator=].
```

Comment:
Are these defaults used anywhere? If not, let's remove them.

Reply:
The defaults are defined so that when creating a new item we don’t have to specify each item explicitly. This is used in the source and trigger JSON parsing.